src/components/editProfessorsModal/professorsList/styles.ts

```diff
@@ -0,0 +1,55 @@
+import { Box, Card, Typography } from '@mui/material';
+import styled from 'styled-components';
+import theme from '../../../utils/theme';
+
+export const Container = styled(Box).attrs({
+  display: 'flex',
+  flexDirection: 'column',
+  gap: '8px',
+  paddingRight: '20px',
+})`
+  overflow: scroll;
```

Comment:
Assim aparece barra de rolagem horizontal mesmo que não tenha o que rolar no eixo X. Para aparecer só na vertical o correto é:

```suggestion
  overflow-y: scroll;
```

| overflow | overflow-y |
| --------- | ----------- |
| ![Captura de Tela 2023-05-02 às 19 16 37](https://user-images.githubusercontent.com/26875510/235805471-d1e6173f-d55f-48a9-9c52-970c2b4ee70c.png) | ![Captura de Tela 2023-05-02 às 19 26 27](https://user-images.githubusercontent.com/26875510/235806289-e7730db1-0208-4491-b06b-6adc794e11f8.png) |